### ISSUE TITLE
Android Java example?

### ISSUE THREAD

**Issue by bobrokrol** [yvanvds/yse-soundengine#12]
looks great, but is there any Java example of how to use this library? I found only this one https://github.com/yvanvds/yse-soundengine/tree/master/android_demos/native but looks like it even do not call any sound API. I'm not Android developer but I need some sound library for for ANE for Adobe AIR so I could miss something.

**Comment by yvanvds** [comment]
This is a native library, intended for native (C++) applications. You can create native applications from scratch with the Android NDK or use other frameworks based on that (like JUCE or most game engines). Java is not used for native development.

**Comment by bobrokrol** [comment]
yes I understand. I mean't Java wrapper and compiled .so. For example like in soundcricket library
http://www.crickettechnology.com/downloads
so it  would have .jar + armeabi-v7a / armeabi .so files. It's everything I need To make ANE. 
I understand that it's possible to compile itself but .

**Comment by yvanvds** [comment]
I see. Well, I do not need a Java wrapper so I doubt I will be creating one myself. I guess it would be possible to wrap the more basic functionality, but you will find that in other sound engines. When it comes to the DSP part of the library, wrapping it would be a very bad idea because it would seriously impact audio performance.

**Comment by bobrokrol** [comment]
Hi, thanks for reply. I thought if it's cross-platform it's ready to use in any Android / IOS projects not in native only. As I said I do not expert in C++ last time I compiled smthing with c++ 5 years ago. I'm searching just simple solution to play sounds in Adobe Air app as their implementation is worst possible. I switched to Android SoundPool but it has critical bugs on Android 5.0... when it's loading sounds extreamely slow(3-5 seconds small sound) and some other bugs. In the Android bug tracker was suggested to use sound cricket lib but it has only two api for playing  
1) stream sounds. Implementation is either buggy or has limitation as my app craches when I try to load more than 10-20 sounds
2) Bank sounds. The problem that such banks cant created from api but I generate thoughants of sounds from some script. And to just test ther lib I need to made many changes in the code but I'm not sure if that lib would be better

So I found this lib and thought it's possible to just test if it would work in my project. Thanks.
